src/onegov/form/parser/core.py

```diff
@@ -798,6 +820,11 @@ class UrlField(Field):
     type: ClassVar[Literal['url']] = 'url'
 
 
+@final
+class VideoURLField(Field):
```

Comment:
Make sure to add `VideoURLField` to the `BasicParsedField` union above, I think you can also change the `ParsedField` union to `BasicParsedField | FileParsedField`, not sure why I haven't done that.